### PR TITLE
[WIP] Adding notebook for Adam Weight decay

### LINKED ISSUE #336: Request for example: Weight Decay Optimizers / Super Convergence
A valuable set of optimizers that are available in Addons.

Related info:
https://www.fast.ai/2018/07/02/adam-weight-decay/
https://github.com/tensorflow/addons/blob/master/examples/README.md

### PR DESCRIPTION
**Pull request by SSaishruthi**
This PR closes #336

### CODE REVIEW

**Comment by SSaishruthi** [comment]
This is WIP. I will update code with more examples

**Comment by gabrieldemarmiesse** [comment]
I'll close this pull request as there was no activity for a while. Feel free to reopen it if necessary.
